src/core/css/SearchResult.scss

```diff
@@ -96,24 +102,28 @@
   display: flex;
 }
 
+.SearchResult-heading-wrapper {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
 .SearchResult-icon {
-  display: inline-block;
-  float: right;
   height: 38px;
+  margin-left: 5px;
   width: 38px;
 
+  [dir=rtl] & {
+    margin: 0 5px 0 0;
```

Comment:
We have a mixin for this in `cire/css/inc/mixins`: `@include margin-start(5px);`